src/tools/run-make-support/src/fs.rs

```diff
@@ -50,7 +14,34 @@ pub fn copy_dir_all(src: impl AsRef<Path>, dst: impl AsRef<Path>) {
             if ty.is_dir() {
                 copy_dir_all_inner(entry.path(), dst.join(entry.file_name()))?;
             } else if ty.is_symlink() {
-                copy_symlink(entry.path(), dst.join(entry.file_name()))?;
+                // Traverse symlink once to find path of target entity.
+                let target_path = std::fs::read_link(entry.path())?;
+
+                let new_symlink_path = dst.join(entry.file_name());
```

Comment:
Is there a reason for this? I'm not sure if this would work properly in all situations. I'd rather just copy the link verbatim.

Reply:
Wait, does `std::fs::copy` simply copy over the link as-is?

Reply:
I don't think `fs::copy` would work. But... oh wait, never mind. I think I got confused by the order of arugments for `symlink_file` again. I keep forgetting they're the reverse of `CreateSymbolicLink`. Ignore me, lol.